daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md

```diff
@@ -400,6 +531,57 @@ You're up and running! Both Dapr and your app logs will appear here.
 
 {{% /codetab %}}
 
+{{% codetab %}}
+
+Transactional state is supported by extending `TransactionalState` base object which hooks into your
+object via setters and getters to provide a transaction. Before we created our own transactional object,
```

Comment:
Please avoid 1st person "we", "our" as described [here](https://docs.dapr.io/contributing/contributing-docs/#style-and-tone)